Formula/mrbayes.rb

```diff
@@ -0,0 +1,32 @@
+class Mrbayes < Formula
+  desc "Bayesian inference of phylogenies and evolutionary models"
+  homepage "https://nbisweden.github.io/MrBayes/"
+  url "https://github.com/NBISweden/MrBayes/archive/v3.2.7a.tar.gz"
```

Comment:
looks like `3.2.7` release is newer than `3.2.7a`?
![image](https://user-images.githubusercontent.com/1580956/131594923-898fecab-52a9-48ed-bde0-e5badd5a36cc.png)

Reply:
https://github.com/NBISweden/MrBayes/releases/tag/v3.2.7a is further along the `develop` branch. The maintainers re-tagged a bunch of old commits when they migrated from SVN.

Reply:
I see, make sense.